corehq/apps/auditcare/tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from django.test import RequestFactory, TestCase
+from testil import Config
+
+from corehq.apps.auditcare.models import NavigationEventAudit
+
+from .test_middleware import make_view
+
+
+class TestNavigationEventAudit(TestCase):
```

Comment:
Can this be `SimpleTestCase`? After `save()` was removed, it doesn't look like this has any interaction with the database.

Reply:
Ah, good point.